proto/executionapis/astria/bundle/v1alpha1/bundle.proto

```diff
@@ -24,6 +24,22 @@ message Bundle {
   bytes prev_rollup_block_hash = 4;
 }
 
+// The AuctionResult message is submitted by the Auctioneer to the rollup as a
+// `RollupDataSubmission` on the sequencer.
+// The rollup will verify the signature and public key against its configuration,
+// then unbundle the body into rollup transactions and execute them first in the
+// block.
+message AuctionResult {
```

Comment:
I have to walk back my suggestion for calling this a `Result` because it would clash with a regular Rust `Result`: `Result<AuctionResult, E>`.

Let's go with `WinningBundle` instead?

Reply:
I think it's probably fine to just call this a SignedBundle? That's what it is and provides a more generic option. IE this could allow it to be a bundle generated and signed by a TEE and submitted.

The fact that this signer indicates it is a Auction Result bundle is an implementation detail for the rollup?

Reply:
After discussion online, I'm ok with `allocation`.

Long term we will want something generic for signed data probably.